internal/controller/postgrescluster/postgres.go

```diff
@@ -621,6 +621,24 @@ func (r *Reconciler) reconcilePostgresDataVolume(
 
 	pvc.Spec = instanceSpec.DataVolumeClaimSpec
 
+	// If a source cluster was provided and VolumeSnapshots are turned on in the source cluster
+	// and there is a VolumeSnapshot available for the source cluster that is ReadyToUse, use it
+	// as the source for the PVC. If there is an error when retrieving VolumeSnapshots, continue
+	// creating PVC in the usual fashion.
```

Comment:
We don't want to surface the error?

Reply:
Maybe we log it, but continue creating the PVC?

Reply:
Works for me.